verify.py

```diff
@@ -354,37 +375,41 @@ def credentials(user):
                                             len(original.graph),
                                             len(destination.graph)
                                             ))
-                
-                # If verbose flag is set, print full resource details to screen
+
+                # If verbose flag is set, print full resource details to screen/file
                 if args.verbose:
-                    print("\nRESOURCE {0}: {1} {2}".format(
+                    logger.info("\nRESOURCE {0}: {1} {2}".format(
                           counter, original.location, original.type
                           ))
-                    print("  rel  =>", original.relpath)
-                    print("  orig =>", original.origpath)
-                    print("  dest =>", original.destpath)
-                    print("  Verifying original to copy... ", end='')
-                    print("{0} -- {1}".format(verified, verification))
-                
+
+                    logger.info("  rel  => {}".format(original.relpath))
+                    logger.info("  orig => {}".format(original.origpath))
+                    logger.info("  dest => {}".format(original.destpath))
+                    logger.info("  Verifying original to copy... {0} -- {1}".format(
+                         verified, verification))
+
+                if not verified:
+                    logger.warn('WARN: Resource Mismatch {}'.format(original.relpath));
+
+
                 # Display a simple counter
-                else:
-                    print("Checked {0} resources...".format(counter), end='\r')
-                
+                print("Checked {0} resources...".format(counter), end='\r')
+
                 # If a CSV summary file has been specified, write results there
                 if args.csv:
-                    row = { 'number':       str(counter), 
+                    row = { 'number':       str(counter),
                             'type':         original.type,
-                            'original':     original.origpath, 
-                            'destination':  original.destpath, 
-                            'verified':     str(verified), 
+                            'original':     original.origpath,
+                            'destination':  original.destpath,
+                            'verified':     str(verified),
                             'verification': verification
                             }
                     writer.writerow(row)
-                    
+
                 counter += 1
-    
+
     # Clear the resource counter display
-    print('')
```

Comment:
Hi Bethany,  this line needs to remain a print('') in order for the counter not to be overwritten when the script finishes.  The desired behavior is that the final count should remain visible when it exits.